src/helpers/parseError.ts

```diff
@@ -0,0 +1,25 @@
+import { isAxiosError } from 'axios'
+
+interface MessageObject {
+  message: string
+}
+
+export function stringifyError(e: unknown) {
+  if (typeof e === 'undefined') return e
```

Comment:
Should return something else

Reply:
If `undefined` we should provide some default text/error, I think